scripts/benchmark.py

```diff
@@ -117,9 +117,9 @@ def execute_task(task: ExecutionTask):
 			print("WARNING: CHECKER FAILED -> DELETING stats!")
 			(result_folder / "stats.yaml").unlink(missing_ok=True)
 
-	vis_script = scripts_path / "visualize.py"
-	for file in visualize_files:
-		run_visualize(vis_script, env, result_folder / file)
+	# vis_script = scripts_path / "visualize.py"
+	# for file in visualize_files:
+	# 	run_visualize(vis_script, env, result_folder / file)
```

Comment:
on purpose?

Reply:
yes, for 3D cases we have meshcat code, which is not integrated into benchmark.py yet. I can add it before merging.